datalad/tests/utils.py

```diff
@@ -469,7 +469,12 @@ def newfunc(*arg, **kw):
         flavors_ = _get_resolved_flavors(flavors)
 
         testrepos_uris = _get_testrepos_uris(regex, flavors_)
-        assert(testrepos_uris)
+        # MIH: I don't think we can assert that there always is a testrepo
+        # for network test when NONETWORK is given, this list will be empty
+        # will change it to skipping the test. Remove on ACK.
+        #assert(testrepos_uris)
```

Comment:
meh - But this means, we have to review the tests rather than adapt herein, I guess.
However, whatever you need atm - just do it ;)